src/strategyHelpers/blueGreen/smiBlueGreenHelper.ts

```diff
@@ -127,7 +127,8 @@ let trafficSplitAPIVersion = ''
 async function createTrafficSplitObject(
    kubectl: Kubectl,
    name: string,
-   nextLabel: string
+   nextLabel: string,
+   annotations: {[key: string]: string} = {}
```

Comment:
how are you letting a user pass this in? You need to add logic to let a user insert it.

We also need to do the same thing for canary.